src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/PersonnelColumns.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { DataTableColumn, useTooltipRef, DoneIcon, WarningIcon, CloseIcon } from '@equinor/fusion-components';
+import Personnel from '../../../../../../models/Personnel';
+
+export type DataItemProps = {
+  item: Personnel;
+};
+
+// TODO: Get proper icons
+const AdStatus = {
+  Available : {text: "Azure AD Approved",icon:<DoneIcon color={"green"}/>},
```

Comment:
Use equinor colors?

Reply:
they are just placeholders till i get the real icons.

Reply:
import { styling } from "@equinor/fusion-components";
styling.colors.green, styling.colors.orange, styling.colors.red <- will give you the equinor/eds hue's